cms/db/submission.py

```diff
@@ -441,8 +441,12 @@ def get_max_evaluation_resources(self):
         t, m = None, None
         if self.evaluated() and self.evaluations:
             for ev in self.evaluations:
-                t = max(t, ev.execution_time)
-                m = max(m, ev.execution_memory)
+                if ev.execution_time is not None \
+                        and (t is None or t < ev.execution_time):
+                    t = ev.execution_time
+                if ev.execution_memory is not None \
+                        and (m is None or m < ev.execution_memory):
+                    m = ev.execution_memory
```

Comment:
Optional: I like the condition more if phrased like this:
```python
if t is None or (ev.execution_time is not None and t < ev.execution_time):
```

Reply:
One reason I prefer the condition I proposed to the one you propose is that yours leads to overwrite `None` with `None`.